docs/search-omero.rst

```diff
@@ -66,6 +66,55 @@ Step-by-Step
 
 #. Click on the Browse link |image2|\ in one search result line of the last image (in the right-hand part of the centre pane) to navigate back to the main webclient.
 
+.. note::
+    Spaces, stars, question marks and ``^`` characters are to be avoided in the Keys.
+    If you already have these characters in Keys in your OMERO, 
+    then try to replace them with underscores if you want to 
+    use the Search functionality on them.
+    Spaces, stars, question marks and ``^`` characters in Values 
+    are acceptable, but should be avoided if possible, see examples below.
+
+Key-Value pairs search examples
+-------------------------------
+
+Considering the following setup of 12 separate images (image 1 through 12):
+
+1. Key ``GFP H2B`` with Value ``2 uM`` on image 1
```

Comment:
This might be more readable in a table of: image id, name, key, value

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/23/commits/86941d46d5c7573855222c80d53df5caae1446fe